### ISSUE TITLE
Fail test if story name contains invalid characters

### ISSUE THREAD

**Issue by DerArvend** [wKich/creevey#11]
Now, if story name contains character, which is not allowed in filename (e.g. `:`), test fails after timeout. In this case it would be more much more convenient if test would fail immediately and with corresponding error

**Comment by MaksimMedvedev** [comment]
Faced the same issue and could not figure out the reason until I went here)
Perhaps I'm kinda blind, but I cannot see any info about forbidden chars in story name. Could you please elaborate on this in docs when you have free time? Thanks for your tremendous work!

**Comment by wKich** [comment]
Yeah, I haven't described it yet. But, sure, I'll add this in docs.
